jquery.xblock.js

```diff
@@ -154,6 +154,15 @@
                     return (lmsBaseURL + '/courses/' + courseId + '/xblock/' + usageId +
                             '/handler/' + handlerName);
                 },
+                local_overrides: {
+                    discussion: function(element, DiscussionUtil) {
+                        var courseId = $(element).data('course-id');
+
+                        DiscussionUtil.localUrls.push('user_profile');
+                        DiscussionUtil.force_async = true;
+                        DiscussionUtil.route_prefix = 'courses/' + courseId + '/discussion'
+                    }
+                }
```

Comment:
Wow, we really shouldn't have any code specific to discussion (or any XBlock really) here, this is a standard library and it breaks the abstraction layer. I imagine this is from another feature and was missed during that code review - can you remove the change from this PR, and put a ticket on top of the backlog to remove the change from the client instance too?